src/citrine/informatics/predictor_evaluation_result.py

```diff
@@ -0,0 +1,257 @@
+from collections import Mapping, Set
+from typing import List, Optional, Type
+from uuid import UUID
+
+from citrine._serialization import properties
+from citrine._serialization.polymorphic_serializable import PolymorphicSerializable
+from citrine._serialization.serializable import Serializable
+from citrine.informatics.predictor_evaluation_metrics import PredictorEvaluationMetric
+from citrine.informatics.predictor_evaluator import PredictorEvaluator
+
+
+class MetricValue(PolymorphicSerializable["MetricValue"]):
+    """[ALPHA] Value associated with a metric computed during a Predictor Evaluation Workflow.
+
+    """
+
+    @classmethod
+    def get_type(cls, data) -> Type[Serializable]:
+        """Return the subtype."""
+        return {
+            "RealMetricValue": RealMetricValue,
+            "PredictedVsActualValue": PredictedVsActualValue,
+        }[data["type"]]
+
+
+class RealMetricValue(Serializable["RealMetricValue"], MetricValue):
+    """[ALPHA] Mean and standard error computed for a real-valued metric
+
+    Parameters
+    ----------
+    mean: float
+        Mean value
+    standard_error: float
+        Standard error of the mean
+
+    """
+    mean = properties.Float("mean")
+    standard_error = properties.Optional(properties.Float(), "standard_error")
+    typ = properties.String('type', default='RealMetricValue', deserializable=False)
+
+    def __init__(self, *,
+                 mean: float,
+                 standard_error: Optional[float]):
```

Comment:
Do we distinguish between `None` and 0? That is, if the argument is None then should we set `self.standard_error` to 0?

Reply:
> None would mean it wasn't computed because there aren't enough samples and 0 would mean that all trials produce the same value

That's correct. Note, we currently set `None` to 0 when we format results on the backend. This is meant to allow us to distinguish between the two scenarios.